cookbook/README.md

```diff
@@ -20,7 +20,8 @@ Note this currently does not include articles/tid-bits on how to use the Flyte p
 1. [Dynamic Tasks](recipes/dynamictasks)
 1. [Tasks without flytekit or Using arbitrary containers](recipes/rawcontainers)
 1. [Jupyter notebook as a task](recipes/notebook_tasks)
-1. [Different container per task](recipes/differentcontainers) 
+1. [Different container per task](recipes/differentcontainers)
+1. [Customize Offloaded Data Location](recipes/offloaded_output_data) 
```

Comment:
instead of this, can we just make this work for all examples?

Reply:
ohh nevermind, I get it now, we are saying that all outputs will be in default storage location and you can customize if you want